### PR TITLE
Cleaning up EndToEnd tests and Sample directories

### PR DESCRIPTION
**Pull request by mathewc**
Because of the recent language restrictions put in place, I can no longer easily run the samples.

I've created a top level language directory for each sample set. I've update the samples E2E tests accordingly. Renamed various functions to remove the "-CSharp" "-Node" suffixes, etc.

### CODE REVIEW

File: sample/.azurefunctions/swagger/swagger.json [hunk 1]
```diff
@@ -1,785 +0,0 @@
-﻿{
-    "swagger": "2.0",
```
Comment (mathewc) [inline sample/.azurefunctions/swagger/swagger.json]: we no longer support swagger

File: test/WebJobs.Script.Tests.Integration/CosmosDB/CosmosDBEndToEndTestsBase.cs [hunk 2]
```diff
@@ -84,12 +85,24 @@ protected async Task CosmosDBTest()
...
     {
         protected CosmosDBTestFixture(string rootPath, string testId, string language) :
-            base(rootPath, testId, language, "Microsoft.Azure.WebJobs.Extensions.CosmosDB", "3.0.0-beta9*")
+            base(rootPath, testId, language)
         {
         }
 
         public DocumentClient DocumentClient { get; private set; }
 
+        protected override ExtensionPackageReference[] GetExtensionsToInstall()
```
Comment (mathewc) [inline test/WebJobs.Script.Tests.Integration/CosmosDB/CosmosDBEndToEndTestsBase.cs]: Hopefully we can get to the point where multiple extensions can be installed. This change will facilitate that. E.g. to enable the rest of the E2E samples for other extension types, we'll need to install multiple extensions.

File: test/WebJobs.Script.Tests.Integration/EventHubs/EventHubSamplesEndToEndTests.cs [hunk 3]
```diff
@@ -1,88 +0,0 @@
...
-using Microsoft.Azure.WebJobs.Script.Config;
-using Microsoft.Azure.WebJobs.Script.Rpc;
-using Microsoft.Extensions.DependencyInjection;
-using Newtonsoft.Json;
-using Newtonsoft.Json.Linq;
-using Xunit;
-
-namespace Microsoft.Azure.WebJobs.Script.Tests.Integration.EventHubs
-{
-    public class EventHubSamplesEndToEndTests : IClassFixture<EventHubSamplesEndToEndTests.TestFixture>
```
Comment (mathewc) [inline test/WebJobs.Script.Tests.Integration/EventHubs/EventHubSamplesEndToEndTests.cs]: Moved this into the Node samples tests

File: test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests_Node.cs [hunk 4]
```diff
@@ -2,39 +2,222 @@
...
-        public SamplesNodeEndToEndTests(TestFixture fixture)
+        public SamplesEndToEndTests_Node(TestFixture fixture)
         {
             _fixture = fixture;
             _settingsManager = ScriptSettingsManager.Instance;
         }
 
-        public object AuthorizationLevelAttribute { get; private set; }
+        [Fact(Skip = "Not currently supported.")]
+        public async Task ServiceBusQueueTrigger_Succeeds()
```
Comment (mathewc) [inline test/WebJobs.Script.Tests.Integration/WebHostEndToEnd/SamplesEndToEndTests_Node.cs]: consolidating these tests here from the other file

File: sample/Bot-CSharp/function.json [hunk 5]
```diff
@@ -1,22 +0,0 @@
-{
-    "bindings": [
```
Comment (mathewc) [inline sample/Bot-CSharp/function.json]: Just deleted the Bot samples w/o migrating. We don't need them anymore

File: sample/Java/HttpTrigger/function.json [hunk 6]
```diff
@@ -1,15 +1,15 @@
...
+    "type" : "httpTrigger",
+    "name" : "req",
+    "direction" : "in",
+    "methods" : [ "get", "post" ]
+  }, {
+    "type" : "http",
+    "name" : "$return",
+    "direction" : "out"
+  } ],
+  "disabled" : false
```
Comment (pragnagopa) [inline sample/Java/HttpTrigger/function.json]: Nit: remove disabled

File: test/WebJobs.Script.Tests.Integration/CosmosDB/CosmosDBEndToEndTestsBase.cs (hunk 2 above)
Comment (pragnagopa) [reply test/WebJobs.Script.Tests.Integration/CosmosDB/CosmosDBEndToEndTestsBase.cs]: 👍

**Pull request merged**